packages/components-date/src/FieldTimeSelect/FieldTimeSelect.stories.tsx

```diff
@@ -36,26 +40,42 @@ export default {
   title: 'Date / FieldTimeSelect',
 }
 
-const Template: Story<FieldTimeSelectProps> = (args) => (
-  <FieldTimeSelect {...args} />
+const Template: Story<FieldTimeSelectProps & { externalLabel: boolean }> = ({
+  externalLabel,
```

Comment:
See proposed revision to `FieldTime` 

```suggestion
  externalLabel = true,
```